src/boundary_conditions.jl

```diff
@@ -136,15 +202,22 @@ function HorizontallyPeriodicBCs(;    top = BoundaryCondition(Flux, nothing),
 end
 
 """
-    ChannelBCs(;  north = BoundaryCondition(Flux, nothing),
-                  south = BoundaryCondition(Flux, nothing),
-                    top = BoundaryCondition(Flux, nothing),
-                 bottom = BoundaryCondition(Flux, nothing))
-
-Construct 'channel' boundary conditions (periodic in ``x``, non-periodic in
-``y`` and ``z``) for ``u`` or a tracer field. The keywords `north`, `south`,
-`top` and `bottom` correspond to boundary conditions in the positive ``y``,
-negative ``y``, positive ``z`, and negative ``z`` directions respectively.
+    ChannelBCs(; north = BoundaryCondition(Flux, nothing),
+                 south = BoundaryCondition(Flux, nothing),
+                   top = BoundaryCondition(Flux, nothing),
+                bottom = BoundaryCondition(Flux, nothing))
+
+Construct `FieldBoundaryConditions` with `Periodic` boundary conditions along the x
+direction. Boundary conditions for the `north` (+y), `south` (-y), `top` (+z), and
+`bottom` (-z) boundaries may be specified. This set of field boundary conditions is
+appropriate for use with a reentrant channel model configuration.
+
+By default no-flux boundary conditions are applied at the `north`, `south`, `top`, and
```

Comment:
The default is shown in the constructor, so it may be best not to supply the redundant information here. For example, the julia docs suggest

> 3. Do not repeat yourself.

Since its may not be clear what a "reentrant channel model configuration" is, it may be best simply to state exactly what the boundary conditions are and leave it at that.